mitodl/ocw-studio
# upgrade ckeditor packages

#### Pre-Flight checklist


#### What are the relevant tickets?

closes #195 

#### What's this PR do?

This upgrades all of our ckeditor packages to `@latest`. I anticipate we'll need to do this again, and it's a pain to ugprade all 20 packages, so I wrote a little nodejs script to do that for us automatically.

#### How should this be manually tested?

The markdown editor should work as before with no errors, in both the minimal and the 'full fat' config. You can test those out on the `/markdown-editor` page.

You can also test out the little upgrade script. If you checkout the `package.json` and `yarn.lock` from `HEAD` on `main` you can then test out upgrading:

```sh
git checkout origin/master -- package.json yarn.lock
node ./scripts/upgrade_ckeditor.js
```

I used the program `commander`, so there is a little help text (`--help`).
